src/api/helpers/decrypt/index.ts

```diff
@@ -0,0 +1,19 @@
+import crypto from 'crypto';
+import dotenv from 'dotenv';
+import { FINANCIAL_DETAILS } from '../../constants';
+import generateKey from '../encrypt/generate-key';
+
+dotenv.config();
+
+const { ENCRYPTION_METHOD, ENCODING, STRING_ENCODING, OUTPUT_ENCODING } = FINANCIAL_DETAILS.ENCRYPTION;
+
+const key = generateKey();
+
+const decrypt = (dataToDecrypt: any) => {
```

Comment:
Can we add some very detailed documentation here, 1) ... 2) ....

Should we split up the methods as well into their own functions (in the same file), so it's easier to read/follow (at least in my mind from what I see)


e.g So this function could be something simpler to read like this:

```js
const buff = createBuffer();
const decipher = createDecipher();
const superAmazingModifiedDecipher = ();
``